projects/js-packages/publicize-components/src/components/social-previews/use-post-data.js

```diff
@@ -28,16 +28,6 @@ export function usePostData() {
 			// If we have a SIG token, use it to generate the image URL.
 			if ( sigImageUrl ) {
 				image = sigImageUrl;
-			} else if ( attachedMedia?.[ 0 ]?.id ) {
-				// If we don't have a SIG image, use the first image in the attached media.
-				const [ firstMedia ] = attachedMedia;
-				const isImage = firstMedia.id
-					? getMedia( firstMedia.id )?.mime_type?.startsWith( 'image/' )
-					: false;
-
-				if ( isImage && firstMedia.url ) {
-					image = firstMedia.url;
-				}
```

Comment:
From now on we never user attached media as link preview

Reply:
You mean that custom media will not be used as OG media?

Reply:
Yes. We decided that OG media is just featured image or SIG for now

Reply:
Cool. Thanks